cms/templates/widgets/footer.html

```diff
@@ -19,11 +19,6 @@
           <li class="nav-item nav-peripheral-pp">
             <a data-rel="edx.org" href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
           </li>
-          % if user.is_authenticated():
-          <li class="nav-item nav-peripheral-feedback">
-            <a href="mailto:courseops@stanfordonline.zendesk.com" rel="external">${_("Contact Us")}</a>
-          </li>
-          % endif
```

Comment:
We already have this link included elsewhere on the page.
Now we have no local changes in the file.